docs/src/routes/docs/components/date-picker.svx

```diff
@@ -0,0 +1,107 @@
+
+<RelatedComponents components={relatedComponents} />
+
+<Showcase>
+  <div slot="showcase" class="padded">
+    <DatePicker format="%m/%d/%Y" />
+    <DatePicker format="%m/%d" noCalendar />
+    <DatePicker range />
+  </div>
+  <div slot="source">
+
+```svelte
+<DatePicker format="%m/%d/%Y" />
+<DatePicker format="%m/%d" noCalendar />
+<DatePicker range />
+```
+
+  </div>
+</Showcase>
+
+
+## Properties {#properties}
+
+### Style Properties
+
+| Name | Default | Type | Description |
+| ---- | ------- | ---- | ----------- |
+| **`noCalendar`** | `false` | `boolean` | Whether to hide the calendar dropdown and only keep the text input. |
+| **`top`** | `false` | `boolean` | Whether to show the calendar dropdown above or below the text field. |
+| **`right`** | `false` | `boolean` | Whether to align the calendar dropdown to the right or the left side of the text field. |
+
+### Functional Properties
+
+| Name | Default | Type | Description |
+| ---- | ------- | ---- | ----------- |
+| **`range`** | `false` | `boolean` | If this is set, the date picker will have two fields – for the start and the end of the date range. |
+| **`value`** | `null` | `Date` or `{ start, end }` | Depending on the value of the ==`range`== prop, this is either a Date object or an object with two fields, `start` and `date`, containing Date objects. |
+| **`locale`** | `undefined` | `string` | The language tag defining the desired locale (e.g., `en-US`). If left `undefined`, the user's locale will be used. |
+| **`firstWeekday`** | `1` | `number` | The index of the weekday to start the week at. <br /> 0 is for Sunday and 6 is for Saturday. |
+| **`format`** | `'%d.%m.%Y'` | `string` | The format string for the text input and representation. The `%`-specifiers are a subset of C date format specifiers, with only `%d`, `%m`, `%y` and `%Y` allowed. |
+
+### Class Properties
+
+All `<DatePicker>` components use the `.date-picker` class.
+
+| Name | Default | Type | Description |
+| ---- | ------- | ---- | ----------- |
+| **`class`** | `null` | `string` | A class string to add to the component. Falsy values disregarded. |
+| **`weekdaysClass`** | `null` | `string` | A class string to the list of weekdays above the calendar. Falsy values disregarded. |
+| **`weekClass`** | `null` | `string` | A class string to add to the element containing each row of days in the calendar. Falsy values disregarded. |
+| **`dayClass`** | `null` | `string` | A class string to add to each day in the calendar. Falsy values disregarded. |
+
+
+## Events {#events}
+
+| Name | Event Detail | Description |
+| ---- | ------------ | ----------- |
+| **`change`** | `{ value }` | Fired when the value of the picker changes. For the range pickers, will only fire when both ends of the range are selected. The `value` field contains the same thing as the `value` property would contain. |
+
+
+## Slots {#slots}
+
+### `between-inputs` slot
```

Comment:
Do we really need the word "slot" after each of them?

Reply:
Yeah, I think it makes it clearer that we're talking about named slots

Reply:
It is already under the "Slots" section, and its name is in a code block. How clearer can it get?

Reply:
(and it generally looks neater, code blocks make for ugly-looking headings